app/services/doc_auth/mock/result_response.rb

```diff
@@ -33,12 +34,12 @@ def errors
         @errors ||= begin
           file_data = parsed_data_from_uploaded_file
 
-          if file_data.blank? || attention_with_barcode?
+          if file_data.blank?
             {}
           else
             doc_auth_result = file_data.dig('doc_auth_result')
             image_metrics = file_data.dig('image_metrics')
-            failed = file_data.dig('failed_alerts')
+            failed = failed_file_data(file_data.dup)
```

Comment:
this field is used to specify failed alert in the test yaml file. This wis duplicating the whole file. If we wanna to overwrite(remove) the default attention with bar code error

we can do like following (specify an empty array):

```
failed_alerts: []
```